guide/04-feature-data-and-analysis/part5_merge_parcels.ipynb

```diff
@@ -84,16 +84,16 @@
   },
```

Comment:
<p>Getting this error:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
Exception                                 Traceback (most recent call last)
~/Job/repos/geosaurus/src/arcgis/gis/__init__.py in _hydrate(self)
  18742             try:
&gt; 18743                 self._refresh()
  18744 

~/Job/repos/geosaurus/src/arcgis/gis/__init__.py in _refresh(self)
  18716                 else:
&gt; 18717                     raise e
  18718 

~/Job/repos/geosaurus/src/arcgis/gis/__init__.py in _refresh(self)
  18705                 else:
&gt; 18706                     dictdata = self._con.post(
  18707                         self.url, params

~/Job/repos/geosaurus/src/arcgis/gis/_impl/_con/_connection.py in post(self, path, params, files, **kwargs)
   1525             return resp
-&gt; 1526         return self._handle_response(
   1527             resp=resp,

~/Job/repos/geosaurus/src/arcgis/gis/_impl/_con/_connection.py in _handle_response(self, resp, file_name, out_path, try_json, force_bytes, ignore_error_key)
   1000                 errorcode = data["error"]["code"] if "code" in data["error"] else 0
-&gt; 1001                 self._handle_json_error(data["error"], errorcode)
   1002             return data

~/Job/repos/geosaurus/src/arcgis/gis/_impl/_con/_connection.py in _handle_json_error(self, error, errorcode)
   1023         errormessage = errormessage + "\n(Error Code: " + str(errorcode) + ")"
-&gt; 1024         raise Exception(errormessage)
   1025 

Exception: User does not have permissions to access 'washingtoncounty.mapserver'.
(Error Code: 403)

During handling of the above exception, another exception occurred:

Exception                                 Traceback (most recent call last)
/var/folders/x2/p8wq0fyj2z37bm9vgf8zbhn80000gp/T/ipykernel_94456/3310369137.py in &lt;module&gt;
----&gt; 1 parcel_fabric_flc = FeatureLayerCollection(service_urls["FeatureServer"], gis)
      2 version_management_server = parcel_fabric_flc.versions
      3 
      4 new_version_name = "merge_version_1"
      5 

~/Job/repos/geosaurus/src/arcgis/features/layer.py in __init__(self, url, gis)
   4331 
   4332         try:
-&gt; 4333             if self.properties.syncEnabled:
   4334                 self.replicas = SyncManager(self)
   4335         except AttributeError:

~/Job/repos/geosaurus/src/arcgis/gis/__init__.py in properties(self)
  18727             return self._lazy_properties
  18728         else:
&gt; 18729             self._hydrate()
  18730             return self._lazy_properties
  18731 

~/Job/repos/geosaurus/src/arcgis/gis/__init__.py in _hydrate(self)
  18764                     # try as a public server
  18765                     self._lazy_token = None
&gt; 18766                     self._refresh()
  18767 
  18768                 except HTTPError as httperror:

~/Job/repos/geosaurus/src/arcgis/gis/__init__.py in _refresh(self)
  18715                     dictdata = self._con.get(self.url, params)
  18716                 else:
&gt; 18717                     raise e
  18718 
  18719         self._lazy_properties = _mixins.PropertyMap(dictdata)

~/Job/repos/geosaurus/src/arcgis/gis/__init__.py in _refresh(self)
  18704                     )
  18705                 else:
&gt; 18706                     dictdata = self._con.post(
  18707                         self.url, params
  18708                     )  # , token=self._lazy_token)

~/Job/repos/geosaurus/src/arcgis/gis/_impl/_con/_connection.py in post(self, path, params, files, **kwargs)
   1524         if return_raw_response:
   1525             return resp
-&gt; 1526         return self._handle_response(
   1527             resp=resp,
   1528             out_path=out_path,

~/Job/repos/geosaurus/src/arcgis/gis/_impl/_con/_connection.py in _handle_response(self, resp, file_name, out_path, try_json, force_bytes, ignore_error_key)
    999                     return data
   1000                 errorcode = data["error"]["code"] if "code" in data["error"] else 0
-&gt; 1001                 self._handle_json_error(data["error"], errorcode)
   1002             return data
   1003         else:

~/Job/repos/geosaurus/src/arcgis/gis/_impl/_con/_connection.py in _handle_json_error(self, error, errorcode)
   1022 
   1023         errormessage = errormessage + "\n(Error Code: " + str(errorcode) + ")"
-&gt; 1024         raise Exception(errormessage)
   1025 
   1026     def post_multipart(

Exception: User does not have permissions to access 'washingtoncounty.mapserver'.
(Error Code: 403)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1685/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
@jyaistMap I don't have control over the enterprise. The services are wiped on some interval. This is fine, I can republish, however, now, the users are no longer able to connect to the server and create content.